### PR TITLE
Boilerplate project starter code for alpha watch face sample.

### PR DESCRIPTION
**Pull request by codingjeremy**
This is boilerplate only for the alpha project. 

I was originally going to develop the Kotlin watch face sample to be the alpha sample, but I decided to separate them, so we have a production version kotlin version as well.

This will be the alpha version and folded into the regular kotlin version (WatchFaceKotlin) once the library exits beta.

### CODE REVIEW

**Review by JolandaVerhoef — APPROVED** [review]

File: WatchFaceAlphaKotlin/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Alpha WatchFace Sample (Kotlin)
+===============================
+Demonstrates watch faces using the new androidx alpha libraries (Kotlin).
```
Comment (tiembo) [inline WatchFaceAlphaKotlin/README.md:3]: androidx => AndroidX

File: WatchFaceAlphaKotlin/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Introduction
+------------
+With the release of the androidx alpha watch face libraries (late 2020), Wear OS has simplified
```
Comment (tiembo) [inline WatchFaceAlphaKotlin/README.md:7]: androidx => AndroidX

File: WatchFaceAlphaKotlin/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+watch face development.
+
+This sample gives you an early preview of how you can simplify your development with the new 100%
+Kotlin libraries.
+
+Steps to build in Android Studio
+--------------------------------
+Once the project is open, edit run configurations -> General -> Launch Options -> Launch: Nothing
+this will allow runs.
```
Comment (tiembo) [inline WatchFaceAlphaKotlin/README.md:16]: clarify "this will allow runs"

File: WatchFaceAlphaKotlin/app/proguard-rules.pro [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 The Android Open Source Project
```
Comment (tiembo) [inline WatchFaceAlphaKotlin/app/proguard-rules.pro:1]: remove file if unused

**Review by tiembo — CHANGES_REQUESTED** [review]
Overall LGTM, some minor nits.

Also, do we want the explicit "alpha" designation in the app name (versus just calling out in the README that alpha libs are being used and naming the app something like "WatchFace (Kotlin)"?

**Review by tiembo — APPROVED** [review]

**Pull request merged**